### ISSUE TITLE
Data Augmentation 

### ISSUE THREAD

**Issue by jalotra** [lamhoangtung/LineHTR#6]
Hey @lamhoangtung , I am making a doctor prescription recogniser. I am using the @githubharald SimpleHTR along with some increased preprocessing steps. The images are word-images and I am currently getting 68 % accuracy on the dataset that I have. I have come through a paper that says that data augmentation will help [paperlink](https://www.semanticscholar.org/paper/Data-Augmentation-for-Recognition-of-Handwritten-a-Wigington-Stewart/b5f8c1a58a390d17c757d33e01edfcf3ff028117), do you have any info if **it would be useful or some tips** what can be done.

P.S I am currently re-training the model with 7 Layered Cnn just like you on IAM and RIMES Dataset firstly and then fine-tuning it on the doctor dataset. Also, I have preprocessed the word images just like @githubharald told.

**Comment by lamhoangtung** [comment]
Comprehensive data augmentation is definitely required to improve the accuracy. I don’t have much trick for you other than:
- Try more comprehensive CNN like ResNet or EfficientNet
- Leverage handwritten font to generate synthesis dataset for training. You would want to start with font first then move to IAM and finally your specifically your real dataset, go from easy looking data to hard ones and make sure to use a really big general corpus and dropout to avoid making that LSTM layer overfit.

Good luck ;)

**Comment by jalotra** [comment]
Okay, thanks for the tip.
